xyzspaces/spaces.py

```diff
@@ -40,9 +40,10 @@
 import geobuf
 import geopandas as gpd
 from geojson import Feature, GeoJSON
+from geopandas import GeoDataFrame
```

Comment:
There is LGTM warning because of multiple imports of geopandas. maybe we can use the same import defined above?

Reply:
Can we?